sysutils/angle-grinder/Portfile

```diff
@@ -0,0 +1,930 @@
+    tokio-uds		0.2.4 99ce87382f6c1a24b513a72c048b2c8efe66cb5161c9061d00bee510f08dc168 \
+    toml		0.4.10 758664fc71a3a69038656bee8b6be6477d2a6c315a6b81f7081f591bffa4111f \
+    try-lock		0.2.2 e604eb7b43c06650e854be16a2a03155743d3752dd1c943f6829e26b7a36e382 \
+    ucd-util		0.1.3 535c204ee4d8434478593480b8f86ab45ec9aae0e83c568ca81abf0fd0e88f86 \
+    unicase		1.4.2 7f4765f83163b74f957c797ad9253caf97f103fb064d3999aea9568d09fc8a33 \
+    unicase		2.2.0 9d3218ea14b4edcaccfa0df0a64a3792a2c32cc706f1b336e48867f9d3147f90 \
+    unicode-bidi		0.3.4 49f2bd0c6468a8230e1db229cff8029217cf623c767ea5d60bfbd42729ea54d5 \
+    unicode-normalization		0.1.7 6a0180bc61fc5a987082bfa111f4cc95c4caff7f9799f3e46df09163a937aa25 \
+    unicode-segmentation		1.2.1 aa6024fc12ddfd1c6dbc14a80fa2324d4568849869b779f6bd37e5e4c03344d1 \
+    unicode-width		0.1.5 882386231c45df4700b275c7ff55b6f3698780a650026380e72dabe76fa46526 \
+    unicode-xid		0.1.0 fc72304796d0818e357ead4e000d19c9c174ab23dc11093ac919054d20a6a7fc \
+    unreachable		1.0.0 382810877fe448991dfc7f0dd6e3ae5d58088fd0ea5e35189655f84e6814fa56 \
+    url		1.7.2 dd4e7c0d531266369519a4aa4f399d748bd37043b00bde1e4ff1f60a120b355a \
+    utf8-ranges		1.0.2 796f7e48bef87609f7ade7e06495a87d5cd06c7866e6a5cbfceffc558a243737 \
+    uuid		0.6.5 e1436e58182935dcd9ce0add9ea0b558e8a87befe01c1a301e6020aeb0876363 \
+    uuid		0.7.1 dab5c5526c5caa3d106653401a267fed923e7046f35895ffcb5ca42db64942e6 \
+    vcpkg		0.2.6 def296d3eb3b12371b2c7d0e83bfe1403e4db2d7a0bba324a12b21c4ee13143d \
+    vec_map		0.8.1 05c78687fb1a80548ae3250346c3db86a80a7cdd77bda190189f2d0a0987c81a \
+    version_check		0.1.5 914b1a6776c4c929a602fafd8bc742e06365d4bcbe48c30f9cca5824f70dc9dd \
+    void		1.0.2 6a02e4885ed3bc0f2de90ea6dd45ebcbb66dacffe03547fadbb0eeae2770887d \
+    walkdir		2.2.7 9d9d7ed3431229a144296213105a390676cc49c9b6a72bd19f3176c98e129fa1 \
+    want		0.0.6 797464475f30ddb8830cc529aaaae648d581f99e2036a928877dfde027ddf6b3 \
+    winapi		0.2.8 167dc9d6949a9b857f3451275e911c3f44255842c1f7a76f33c55103a909087a \
+    winapi		0.3.6 92c1eb33641e276cfa214a0522acad57be5c56b10cb348b3c5117db75f3ac4b0 \
+    winapi-build		0.1.1 2d315eee3b34aca4797b2da6b13ed88266e6d612562a0c46390af8299fc699bc \
+    winapi-i686-pc-windows-gnu		0.4.0 ac3b87c63620426dd9b991e5ce0329eff545bccbbb34f3be09ff6fb6ab51b7b6 \
+    winapi-util		0.1.1 afc5508759c5bf4285e61feb862b6083c8480aec864fa17a81fdec6f69b461ab \
+    winapi-x86_64-pc-windows-gnu		0.4.0 712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f \
+    wincolor		0.1.6 eeb06499a3a4d44302791052df005d5232b927ed1a9658146d842165c4de7767 \
+    wincolor		1.0.1 561ed901ae465d6185fa7864d63fbd5720d0ef718366c9a4dc83cf6170d7e9ba \
+    ws2_32-sys		0.2.1 d59cefebd0c892fa2dd6de581e937301d8552cb44489cdff035c6187cb63fa5e \
+    xattr		0.2.2 244c3741f4240ef46274860397c7c74e50eb23624996930e484c16679633a54c \
+    zip		0.5.0 00acf1fafb786ff450b6726e5be41ef029142597b47a40ce80f952f1471730a0
```

Comment:
Are tabs mixed in here? Please use spaces and align the version and checksum columns.

Reply:
Changes made